Tests/Types/HttpHeaderTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * This file is part of DoctrineRestDriver.
+ *
+ * DoctrineRestDriver is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * DoctrineRestDriver is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with DoctrineRestDriver.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace Circle\DoctrineRestDriver\Tests\Types;
+
+use Circle\DoctrineRestDriver\Types\HttpHeader;
+use PHPSQLParser\PHPSQLParser;
+/**
+ * Tests the curl options
+ *
+ * @author    Tobias Hauck <tobias@circle.ai>
+ * @copyright 2015 TeeAge-Beatz UG
+ *
+ * @coversDefaultClass Circle\DoctrineRestDriver\Types\CurlOptions
+ */
+class HttpHeaderTest extends \PHPUnit_Framework_TestCase {
+
+    /**
+     * @var array
+     */
+    private $options;
+
+    /**
+     * @var array
+     */
+    private $expected;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setUp() {
+        $this->options = [
+            'security_strategy'  => 'none',
+            'CURLOPT_MAXREDIRS'  => 22,
+            'CURLOPT_HTTPHEADER' => 'Content-Type: text/plain'
+        ];
+
+        $this->expected = [
+            'CURLOPT_HTTPHEADER'=> ['Content-Type: text/plain']
+        ];
+    }
+
+    /**
+     * @test
+     * @group  unit
+     * @covers ::create
+     *
+     * @SuppressWarnings("PHPMD.StaticAccess")
+     */
+    public function create() {
+        $query    = 'SELECT name FROM products WHERE id=1';
```

Comment:
May you be so kind to align all equal signs? :S Sorry for being a perfectionist, but I truly believe in perfectionism as the only way to get rid of bugs